tests/unit/coordinator_instance.cpp

```diff
@@ -34,18 +34,23 @@ using memgraph::io::network::Endpoint;
 
 class CoordinatorInstanceTest : public ::testing::Test {
  protected:
-  void SetUp() override {}
+  void SetUp() override {
+    if (!std::filesystem::exists(main_data_directory)) return;
+    std::filesystem::remove_all(main_data_directory);
+  }
 
-  void TearDown() override {}
+  void TearDown() override {
+    if (!std::filesystem::exists(main_data_directory)) return;
+    std::filesystem::remove_all(main_data_directory);
```

Comment:
are these all unit tests on which we need to delete data dir

Reply:
Yes, only four of them